File: Day 16/Aayushi-Mittal.py
# To find whether a number is Palindrome or not.

rev=0
num = int(input())
i = num
while i!=0 :
    rev = rev*10 + (i%10)
    i=i/10
if num == rev:
    print(num," is a palindrome.")
else:
    print(num," is not a palindrome.")

File: Day 16/Aayushi-Mittal.cpp
// To find whether a number is Palindrome or not.

#include<iostream>
using namespace std;
int main()
{
  int num, rev=0;
  cin>>num;
  for(int i=num; i!=0; i=i/10)
  {
    rev = rev*10 + (i%10);
  }
  if (num == rev)
    cout<<num<<" is a palindrome.";
  else
    cout<<num<<" is not a palindrome.";
}
